### ISSUE TITLE
Auto-fetch next segment not working

### ISSUE THREAD

**Issue by ragtux** [dbeaver/dbeaver#15750]
#### System information: 
- Operating system: Windows 10
- 22.0.0.202203060510

#### Connection specification:
- PostgreSQL version 14
- PostgreSQL JDBC Driver

#### Describe the problem you're observing:
Auto-fetch next segment is not working even though box is checked. Should auto-fetch when scrolled down entirely in results tab. 

![image](https://user-images.githubusercontent.com/56554701/157138962-d6a7917f-5e8f-4b07-ad50-e4f79830f032.png)
 
In text view there is also a popup:
![image](https://user-images.githubusercontent.com/56554701/157153812-939416ce-483a-4ed7-ae2d-914f08eed74b.png)

**Comment by ragtux** [comment]
It seems like auto-fetch has a problem when the table has fewer records in it (~10,000) it appears to work on tables with more rows than this.

**Comment by serge-rider** [comment]
Could you past a sample SQL query you execute?

In 22.0 we added scroll limitation for queries which may potentially update table data (e.g. UPDATE, etc).

**Comment by ragtux** [comment]
huh, I am now unable to replicate this behavior on my end. It seems to be working normally. The query was:
```
SELECT * 
FROM tbl 
ORDER BY id
```
I tried restarting dbeaver and rebooting my machine but it still wasn't working. I think I had the result-set fetch size set to 1000 and I tried various fetch sizes. 

I will close this issue as I cannot replicate this behavior. 

EDIT: I had just upgraded to the latest version (22.0.0), not sure if that may have played a role in it.
